radar-auth/src/main/java/org/radarcns/auth/authorization/RadarAuthorization.java

```diff
@@ -26,11 +26,14 @@
 
     /**
      * Check if the user authenticated with the given token has the given permission. Not taking
-     * into account project affiliations.
+     * into account project affiliations. Throws a {@link NotAuthorizedException} if the supplied
```

Comment:
This last sentence would not be necessary, it is documented with the `@throws` tag.